### ISSUE TITLE
Faulty "Jack Be Nimble, Jack Be Thick" due to CSV conversion 

### ISSUE THREAD

**Issue by wli-linda** [gucci-on-fleek/Stardew-Valley-Item-Finder#7]
### Describe the bug
I noticed that the alignment of the "Jack Be Nimble, Jack Be Thick" item was off in the display. This is likely due to the comma used in the name of this item.
![image](https://github.com/user-attachments/assets/16d1a7e1-9937-4223-a94e-f8899f7e29fd)

In the .csv:
![image](https://github.com/user-attachments/assets/a4116b3b-3a42-49fc-b43d-017a7b6ace54)

### Browser and OS
Chrome & Windows but I assume it's a setup agnostic bug 

### Save file
Would prefer not to provide it unless it's hard to replicate otherwise

**Comment by gucci-on-fleek** [comment]
@wli-linda

> ### Describe the bug
> 
> I noticed that the alignment of the "Jack Be Nimble, Jack Be Thick" item was off in the display. This is likely due to the comma used in the name of this item. 

Yes, that is correct. When I first made this tool, none of the items had commas in their names, so I never thought to test against that.

> ### Browser and OS
> 
> Chrome & Windows but I assume it's a setup agnostic bug

Yes, correct again.

> ### Save file
> 
> Would prefer not to provide it unless it's hard to replicate otherwise

Fair enough. I don't specifically need _your_ save file, but the newest save file that I have is for v1.5.1, so I won't be able to test the fix against this exact item.

---

Anyways, I _think_ that I've fixed this in 375d354e884b61293dcde3bc5faeb7c972b4a557, so it should hopefully work now. The tool uses pretty aggressive caching, so it might take up to 24 hours for the fix to show up. But if it still doesn't work, just let me know and I'll try something else.

And thanks for the bug report!
